content/blog/2022-06-14-DVC-VS-Code-extension.md

```diff
@@ -0,0 +1,158 @@
+descriptionLong: >
+  VS Code has become the IDE of choice for many data scientists. Today we are
+  releasing the DVC extension, which brings a full ML experimentation platform
+  to VS Code. Manage your data, run experiments, compare metrics, and visualize
+  plots, all from the comfort of your IDE.
+picture: 2022-06-14/header.png
+author: rob_dewit
+# commentsUrl:
+tags:
+  - VS Code
+  - DVC
+  - MLOps
+  - Release
+---
+
+Since its beta release in 2017, DVC has become an essential tool for many data
+science teams. Its data versioning capabilities, reproducible pipelines, and
+experiment tracking features are at the core of our ecosystem of open MLOps
+tools.
+
+Today we are proud to launch a new product that extends how ML teams can use
+DVC: our extension for Visual Studio Code.
+
+With this extension you get a full VS Code-native experimentation platform for
+your machine learning projects. You can control datasets, run experiments, view
+metrics, create plots, and much more. All in one place in your IDE.
+
+[Install the DVC extension from the marketplace to get started.](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
+
+# Why an extension?
+
+We built DVC to expand upon the Git workflow. This approach has made it a
+powerful tool that can easily be picked up by users familiar with Git. However,
+it can be more difficult for those less familiar with CLI tools to get
+accustomed to using DVC. Moreover, while DVC provides many useful
+visualizations, a CLI tool can only take things so far.
+
+Therefore we wanted to incorporate DVC's functionality in a graphical interface.
+[VS Code is the IDE of choice for many](https://insights.stackoverflow.com/survey/2021#section-most-popular-technologies-integrated-development-environment),
+including most of us here at Iterative. It seemed like the natural choice for a
+platform to make DVC more versatile, accessible, and useful.
+
+With this extension, we want to achieve the following:
+
+- Move the entire data science workflow into your IDE
+- Enhance existing visualizations and provide new ones
+- Make DVC more accessible by hiding the complexity of the CLI
+
+As data scientists, DVC is our toolbox. This extension turns VS Code into our
+workshop.
+
+<!-- # Why Visual Studio Code
+
+One of [our core beliefs at Iterative](https://iterative.ai/why-iterative/) is
+that our products should couple tightly with existing tools and processes for
+data science projects.
+[VS Code is the IDE of choice for many](https://insights.stackoverflow.com/survey/2021#section-most-popular-technologies-integrated-development-environment),
+including many of us at Iterative. Here are a few reasons why we really like it:
+
+VS Code...
+
+- is open source and lightweight
+- has excellent support for Git workflows
+- supports Jupyter Notebooks
+- comes with remote development and execution built-in
+- has a rich ecosystem of extensions -->
+
```

Comment:
Yep, prob. unnecessary 🙂 IMO it's already clear enough from the popularity mention above.

```suggestion
```